platform/linux-generic/arch/arm/odp/api/abi/cpu.h

```diff
@@ -20,7 +20,8 @@ static inline void odp_cpu_pause(void)
 	 * ISB flushes the pipeline, then restarts. This is guaranteed to
 	 * stall the CPU a number of cycles.
 	 */
-	__asm volatile("isb" ::: "memory");
+	__asm volatile("isb sy" ::: "memory");
```

Comment:
commit description does not match code.

Reply:
yea, I tried to use _buidin but this version of clang does not support it.